pycbc/inference/io/posterior.py

```diff
@@ -34,7 +34,7 @@ class PosteriorFile(BaseInferenceFile):
 
     def read_raw_samples(self, fields, **kwargs):
         samples = self[self.samples_group]
-        return {field: samples[field][:] for field in fields}
+        return {field: samples[field][:] for field in samples}
```

Comment:
I agree with Collin. I had recently fixed this to use fields, as it was otherwise ignoring the "fields" option (see PR #2514). Is there a particular reason to move back to "samples"? If so, then "fields" should be removed from the function's input parameters...